src/main/java/de/hysky/skyblocker/skyblock/dungeon/terminal/TerminalSolver.java

```diff
@@ -0,0 +1,10 @@
+package de.hysky.skyblocker.skyblock.dungeon.terminal;
+
+import de.hysky.skyblocker.config.SkyblockerConfigManager;
+
+public interface TerminalSolver {
```

Comment:
Very confusing name imo. It's not what it does at all. Should just be a method in `ContainerSolver`. No need for this separate class.

Reply:
How is it confusing? It's an interface that all terminal solvers implement as a way to easily check for click blocking. The blocking click option is specific to terminals if u looked in the config so a global method makes zero sense.

Reply:
Nvm misunderstood a bit.